### ISSUE TITLE
Loss is NAN

### ISSUE THREAD

**Issue by shihaoqing** [hkzhang91/HiNAS#5]
After running the 03_search_CR_R0.yaml, the genotype of the best supernet accuracy is Val: SSIM:0.615 PSNR:23.8382. However，When I train the net by the genotype. The loss function does not converge and sometimes it is nan. I don't change the code.

The follow is the train.yaml file
DATASET:
  DATA_ROOT: 
  DATA_NAME: BSD500_300
  CROP_SIZE: 64
  TASK: "dn"
  LOAD_ALL: True
  TO_GRAY: True
SEARCH:
  TIE_CELL: False
INPUT:
  CROP_SIZE_TRAIN: 64
SOLVER:
  TRAIN:
   MAX_ITER: 600000
  CHECKPOINT_PERIOD: 1000  
  VALIDATE_PERIOD: 1000
  LOSS: ['mse', 'log_ssim']
  LOSS_WEIGHT: [1.0, 0.6]
DATALOADER:
  NUM_WORKERS: 2
  BATCH_SIZE_TRAIN: 24
  BATCH_SIZE_TEST: 24
  SIGMA: [30]
  DATA_AUG: 5
MODEL:
  FILTER_MULTIPLIER: 20
  META_ARCHITECTURE: Dn_compnet
  META_MODE: Width
  NUM_STRIDES: 3
  NUM_LAYERS: 3
  NUM_BLOCKS: 4
  IN_CHANNEL: 1
  PRIMITIVES: "NO_DEF_L"
  ACTIVATION_F: "Leaky"
  USE_ASPP: False
  USE_RES: True

OUTPUT_DIR: output_R0

l have trid to use learning rate = 0.01. The loss is small but still does not converge
